src/ducks/entities/index.js

```diff
@@ -0,0 +1,38 @@
+import { reduceArrayToObj } from 'ducks/normalizer'
+import _ from 'lodash'
+
+export const types = {
+  SENDING_REQUEST: 'ENTITIES/SENDING_REQUEST',
+  REQUEST_ERROR: 'ENTITIES/REQUEST_ERROR',
+  CLEAR_ERROR: 'ENTITIES/CLEAR_ERROR',
+  SET: 'ENTITIES/CLEAR_ERROR',
+}
+
+export const actions = {
+  sendingRequest: (sending: boolean) => ({ type: types.SENDING_REQUEST, sending }),
+  requestError: (error: string) => ({ type: types.REQUEST_ERROR, error }),
+  clearError: () => ({ type: types.CLEAR_ERROR }),
+  setEntities: (nodes) => ({ type: types.SET, nodes }),
+}
+
+export const mergeEntities =
+  (state = {}, action) => (action.nodes ? _.merge({}, state, action.nodes) : state)
```

Comment:
No, we can't, it doesn't do deep merge and will replace object properties with the new object, look at the difference http://codepen.io/mkate/pen/rWQrKO?editors=1111, as long as we do normalized state we shouldn't use `Object.assign` for entities here. But `Object.assign` can be used in other reducers where we expect this behaviour.